### PR TITLE
docs: update readme [skip ci]

### PR DESCRIPTION
**Pull request by pengjiang80**
Update the readme document to add Why Walrus and feature introduction in English version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +16,24 @@
 Walrus is an open-source application management platform that simplifies application deployment and management on any infrastructure. 
 It helps platform engineers build golden paths for developers and empowers developers with self-service capabilities.
 
+# Why Walrus
+
+- An enterprise level XaC (Everything as Code) Platform that enables DevOps collaboration and self-service.
```
Comment (gitlawr) [inline README.md:21]: s/enterprise level/enterprise-level/

File: README.md [hunk 2]
```diff
@@ -10,6 +16,24 @@
...
+- Leverage existing tools including Terraform and OpenTofu, with a more powerful and flexible orchestration and abstraction.
```
Comment (gitlawr) [inline README.md:22]: remove `a`

File: README.md [hunk 3]
```diff
@@ -10,6 +16,24 @@
 Walrus is an open-source application management platform that simplifies application deployment and management on any infrastructure. 
 It helps platform engineers build golden paths for developers and empowers developers with self-service capabilities.
 
+# Why Walrus
+
+- An enterprise level XaC (Everything as Code) Platform that enables DevOps collaboration and self-service.
+- Leverage existing tools including Terraform and OpenTofu, with a more powerful and flexible orchestration and abstraction. 
+- Without the need to be experts in Kubernetes or infastructure, Developers can deploy and manage the entire application system on any infrastructure.
```
Comment (gitlawr) [inline README.md:23]: s/infastructure/infrastructure/

File: README.md [hunk 4]
```diff
@@ -10,6 +16,24 @@
...
+
+- An enterprise level XaC (Everything as Code) Platform that enables DevOps collaboration and self-service.
+- Leverage existing tools including Terraform and OpenTofu, with a more powerful and flexible orchestration and abstraction. 
+- Without the need to be experts in Kubernetes or infastructure, Developers can deploy and manage the entire application system on any infrastructure. 
+
+
+# Key Features
+
+- Unified Orchestration ：Cloud-Native applications are more than Kubernetes. Walrus orchestrates the entire application system, encompassing both application services and resource dependencies (such as databases, middleware, load balancers, and networks).
+- Seperation of Concerns : By leveraging [resource definitions](https://seal-io.github.io/docs/operation/resource-definition) provided by ops team, dev team can define resource type requirements and acheive self-service.
```
Comment (gitlawr) [inline README.md:29]: s/Seperation/Separation/

File: README.md [hunk 5]
```diff
@@ -10,6 +16,24 @@
...
+# Why Walrus
+
+- An enterprise level XaC (Everything as Code) Platform that enables DevOps collaboration and self-service.
+- Leverage existing tools including Terraform and OpenTofu, with a more powerful and flexible orchestration and abstraction. 
+- Without the need to be experts in Kubernetes or infastructure, Developers can deploy and manage the entire application system on any infrastructure. 
+
+
+# Key Features
+
+- Unified Orchestration ：Cloud-Native applications are more than Kubernetes. Walrus orchestrates the entire application system, encompassing both application services and resource dependencies (such as databases, middleware, load balancers, and networks).
```
Comment (gitlawr) [inline README.md:28]: s/：/:/

**Comment by thxCode** [comment]
the commit message adopts https://www.conventionalcommits.org/en/v1.0.0/, please rebase the commits and prepend a `docs: ` prefix. 

FYI, https://github.com/search?q=repo%3Aseal-io%2Fwalrus+docs&type=commits.

**Pull request closed without merging**